source/lib/local-store.js

```diff
@@ -0,0 +1,10 @@
+export default {
+	async get(key) {
+		const result = await browser.storage.local.get([key]);
+		return result[key];
+	},
+
+	async set(key, value) {
+		await browser.storage.local.set({key: value});
```

Comment:
Did you mean `{ [key]: value }`?

Reply:
According to https://developer.chrome.com/extensions/storage, this is the format.

But I think the format you specified works too, for using the same value for multiple keys. 🤷

Reply:
this is if you literally want `"key"` as the key. but since you're passing `key` as param, you probably want computed property